docs/minikube_start.md

```diff
@@ -19,6 +19,7 @@ minikube start
       --disk-size=20g: Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g)
       --docker-env=[]: Environment variables to pass to the Docker daemon. (format: key=value)
       --iso-url="https://storage.googleapis.com/minikube/minikube-0.5.iso": Location of the minikube iso
+      --kubernetes-version="v1.3.0": The kubernetes version that the minikube VM will run
```

Comment:
The v prefix is everywhere else when dealing with Kubernetes versions, so I'd prefer to have it